Code/Framework/AzFramework/Platform/Linux/AzFramework/Application/Application_Linux.cpp

```diff
@@ -16,17 +16,30 @@ namespace AzFramework
     class ApplicationLinux
         : public Application::Implementation
         , public LinuxLifecycleEvents::Bus::Handler
+#if AZ_LINUX_WINDOW_MANAGER_XCB
+        , public LinuxXCBConnectionManager::Bus::Handler
+#endif // AZ_LINUX_WINDOW_MANAGER_XCB
     {
     public:
         ////////////////////////////////////////////////////////////////////////////////////////////
         AZ_CLASS_ALLOCATOR(ApplicationLinux, AZ::SystemAllocator, 0);
         ApplicationLinux();
         ~ApplicationLinux() override;
 
+#if AZ_LINUX_WINDOW_MANAGER_XCB
+        virtual xcb_connection_t* GetXCBConnection() const override;
+#endif // AZ_LINUX_WINDOW_MANAGER_XCB
+        
+
         ////////////////////////////////////////////////////////////////////////////////////////////
         // Application::Implementation
         void PumpSystemEventLoopOnce() override;
         void PumpSystemEventLoopUntilEmpty() override;
+    private:
+#if AZ_LINUX_WINDOW_MANAGER_XCB
+        xcb_connection_t    * m_xcbConnection = NULL;
```

Comment:
`nullptr` should be used for this.
Also the pointer should be attached to `xcb_connection_t`.

Reply:
N/A, but done in the new impl class